docs/_articles/en/v3.x/salesforce-graph-engine/rules.md

```diff
@@ -5,13 +5,13 @@ lang: en
 
 Salesforce Graph Engine includes path-based and data-flow analysis rules.
 
-| Rule | Type | Description |
-| -------- | ----------- | ----------- |
-| ApexFlsViolationRule | Path-based analysis | Detects Create, Read, Update, and Delete (CRUD) and Field-Level Security violations. |
-| ApexNullPointerExceptionRule | Path-based analysis | Identifies Apex operations that dereference null objects and throw NullPointerExceptions. |
-| MultipleMassSchemaLookupRule | Path-based analysis | Detects scenarios where expensive schema lookups are made more than one time in a paths. |
-| UnimplementedTypeRule | Graph-based analysis | Detects abstract classes and interfaces that are non-global and missing implementations or extensions. |
-| UnusedMethodRule (pilot) | Path-based analysis | Detects methods contained in your code that aren’t invoked from any entry points that Graph Engine recognizes. |
+| Rule | Type | Status | Description |
+| -------- | ----------- | ----------- | ----------- |
+| ApexFlsViolationRule | Path-based analysis | Generally Available (GA) | Detects Create, Read, Update, and Delete (CRUD) and Field-Level Security violations. |
+| ApexNullPointerExceptionRule | Path-based analysis | GA | Identifies Apex operations that dereference null objects and throw NullPointerExceptions. |
+| MultipleMassSchemaLookupRule | Path-based analysis | Pilot | Detects scenarios where expensive schema lookups are made more than one time in a paths. |
+| UnimplementedTypeRule | Graph-based analysis | GA | Detects abstract classes and interfaces that are non-global and missing implementations or extensions. |
+| UnusedMethodRule | Path-based analysis | Pilot | Detects methods contained in your code that aren’t invoked from any entry points that Graph Engine recognizes. |
 
 ## ApexFlsViolationRule#
```

Comment:
Not sure how I missed it for so long, but this doesn't need a trailing `#`. Nor does the header for the Null pointer exception rule.